flang/docs/Intrinsics.md

```diff
@@ -847,7 +847,7 @@ used in constant expressions have currently no folding support at all.
 
 #### Usage and Info
 
-- **Standard:** Fortran 2008 and later, specified in 16.9.73
+- **Standard:** Fortran 2008 and later, specified in Section 16.9.73
```

Comment:
The standard uses the word "subclause", not "section".

Reply:
done